content/setting-up-services/how-to-set-up-services/running-a-background-process.md

```diff
@@ -40,3 +48,38 @@ services:
           - echo "npm start --prefix ${TUGBOAT_ROOT}" >> /etc/service/node/run
           - chmod +x /etc/service/node/run
 ```
+
+## Use the ready command
```

Comment:
This is all good information about the ready command, but this doesn't start a background process.

The `start` command should be used for that instead.

Reply:
Hah. Apologies. I conflated my notes. I'll move this to the service commands section, and expand `start` here.